release-notes/docs/3.7.0.md

```diff
@@ -0,0 +1,182 @@
+- [And more...](#and-more)
+
+## Known issues
+
+Not fully supported by `medic-conf@2.x`. For full functionality, users must upgrade to `medic-conf@3.x`.
+
+## Upgrade notes
+
+### Breaking changes
+
+A new OS level dependency (`xsltproc`) is required for generating XFORM `html` on the server.  
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Configurable hierarchies
+
+It is now possible to configure contact hierarchies.
+This allows for better tailored solutions for new projects and opens up new configuration 
+possibilities: for example having any number of place levels, multiple person types, or unbalanced place trees. 
+The previously hardcoded contact hierarchy has now become the default hierarchy and 
+existing projects will continue using it. For more information, please read the
+[configuration information](https://github.com/medic/medic-docs/blob/master/configuration/app-settings.md#configuring-the-contact-hierarchy).
+
+## Includes a new default configuration
+
+Version 3.7.0 includes a complete default configuration that is enabled automatically 
+on fresh application installations. The configuration is designed as a reference application 
+for antenatal care. It provides a foundation for forms, data fields, 
+and analytics to model an ideal way of structuring and organizing 
+a workflow, its configuration code, and testing framework.
+
+## Optimised form HTML generation
+
+Generating the HTML for XForms is moved from being executed on every device to 
+being executed once on the server. 
+This reduces the time to initially load forms, along with reducing the memory 
+footprint of the app by not caching every form's `html` and by not storing the additional files
+required for generating it. 
+
+## Improved purging now done server-side
+
+Marking documents as no longer useful so that they don't take up mobile device resources 
+is now handled server-side.
+This enables devices to not download unnecessary documents during an initial replication 
+and makes the actual "purging" process faster. 
+For more information about changes in purging, please refer to the [purging documentation](https://github.com/medic/medic-docs/blob/master/configuration/purging-serverside.md).
+
+## Warn when replicating too many docs
+
+A warning is now displayed when a user would download more than the recommended maximum of 
+10.000 documents. This warning is shown before starting initial replication in an attempt 
+to reduce the incidence of misconfigured users.
+
+![Warn when replicating too many docs](images/3.7.0-replication-warning.png)
+
```

Comment:
There are a whole heap of performance improvements and it appears that they have noticeably improved certain metrics. Consider including a section summarising the improvements with a graph to show what users can expect!